packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import { useLocation } from 'react-router-dom';
-import MT5AccountNeededModal from 'App/Components/Elements/Modals/mt5-account-needed-modal.jsx';
```

Comment:
Looks like we were importing `MT5AccountNeededModal` only in here. 
If we are removing this usage, maybe we can get rid of the file too.

Reply:
done, thanks to @nikita-deriv